### PR TITLE
add 30s delay only for the first loader

### PR DESCRIPTION
**Pull request by roydahan**

### CODE REVIEW

**Comment by roydahan** [comment]
I haven't tested it yet.
I believe it should solve the issue we had in longevity.

File: sdcm/cluster.py [hunk 1]
```diff
@@ -1785,6 +1786,9 @@ def node_run_stress(node, loader_idx, cpu_idx, keyspace_idx, profile, stress_cmd
                                                           cpu_idx, ks_idx, profile, stress_cmd))
                     setup_thread.daemon = True
                     setup_thread.start()
+                    if first_loader:
```
Comment (amoskong) [inline sdcm/cluster.py]: We should sleep for creating all keyspaces, not just for first keyspace.

`if loader_idx == 0 and cpu_idx == 0:`

Comment (roydahan) [reply sdcm/cluster.py]: You are right.
First, I didn't even think of the multiple keyspaces.
Second, we have the loader_idx and cpu_idx, so I could use it...

**Comment by roydahan** [comment]
@amoskong, please backport to 2.0 and 2.1

**Comment by roydahan** [comment]
I backported it to branch-2.0

**Comment by amoskong** [comment]
I suggested to backport patches by `git cherry-pick -x .....`, we can easy
to find the source commit.

       -x

           When recording the commit, append a line that says "(cherry
picked from commit ...)" to the original commit message in order to
indicate which commit this

           change was cherry-picked from. This is done only for cherry
picks without conflicts. Do not use this option if you are cherry-picking
from your private branch

           because the information is useless to the recipient. If on the
other hand you are cherry-picking between two publicly visible branches
(e.g. backporting a fix

           to a maintenance branch for an older release from a development
branch), adding this information can be useful.

**Pull request merged**